src/pages/guides/develop/coi.md

```diff
@@ -0,0 +1,143 @@
+
+Please note that this change affects *all* add-ons, even those not published in the add-on marketplace (i.e., private and internally distributed add-ons).
+
+Currently, this change *only impacts* Chromium-based browsers (e.g., Chrome, Edge, Opera, etc.). Firefox and Safari browsers are currently exempt. If you've developed a an add-on with mobile support, this would apply to add-ons running on Android devices as well.
+
+## Types of failures
+
+Some failures will be more evident than others, but all will negatively impact the user experience of your add-on.
+
+- If a nested iframe fails to load, Chrome displays a very obvious error message inside the iframe indicating that the domain "refused to connect".
+- If an image fails to load, you may notice missing images in your add-on’s user interface. You should also see failures in the Network section of the browser’s developer tools.
+- If a network call fails due to JavaScript code, you should see warning and error messages in the browser’s developer tools.
+
+## Testing your add-on
+
+Until Adobe Express enables these headers by default, you must configure your local development environment to simulate these changes.
+
+### Apply local header overrides
+
+The developer tools in Chromium-based browsers on the desktop allow you to specify header overrides. Using this, you can simulate the headers that Adobe Express will enable to test whether your add-on has any problems in this environment.
+
+<InlineAlert slots="header, text" variant="info"/>
+
+Mobile add-ons
+
+You cannot test this on mobile devices. You should test your add-on on a desktop web browser. Any issues you run into would also appear on mobile devices, and any fixes you apply would also apply to mobile users.
+
+To enable this environment yourself, perform the following steps:
+
+First, launch your browser’s developer tools and navigate to the "Network" tab, and then navigate to [Adobe Express](https://new.express.adobe.com) in the browser. Your network panel should fill up with a lot of network traffic, like this:
+
+![Network panel screenshot](./img/coi-test-1.png)
+
+To apply an override, right click on the entry for "new.express.adobe.com" and select **Override Headers**.
+
+![Override headers screenshot](./img/coi-test-2.png)
+
+Assuming you haven’t done this before, the developer tools will ask you to pick a folder on your local file system where these overrides are stored. The alert is easy to miss, since it doesn’t present as a dialog box, but rather a message near the top of your developer tool window.
```

Comment:
Can we call this out using an alert block?

Reply:
> Can we call this out using an alert block?

I tried it but it threw off the whole thing with steps due to the fact that alerts always have to be left-aligned (no indent), but the steps around it create indent so it looked strange. I marked it with **IMPORTANT:** instead for now.